### PR TITLE
Remove production warning about prisma from readme

### PR DESCRIPTION
**Pull request by jtoar**
It's out of beta now!

Via the redwoodjs/redwood repo: https://github.com/redwoodjs/redwood/issues/688.

### CODE REVIEW

**Comment by mojombo** [comment]
Only Prisma 2 Client, though. We should still mention that Migrate is not production-ready.

**Comment by thedavidprice** [comment]
I'm attempting to work up a draft doc about overall "readiness" -- will submit for review once I'm done with a first pass.

**Comment by jtoar** [comment]
@thedavidprice Updated the language to distinguish between the readiness of Prisma2 Client and Migrate. Did you want to link to the redwood doc on readiness here too? Otherwise, this is just a small language change to reflect the resounding "yes" you get when you visit isprisma2ready.com.

**Comment by thedavidprice** [comment]
Ok, there are several things going on here at once, all interrelated. I'll outline them and then you can decide what you do/don't want to tackle now.

**1. Two CRA README's**
The file `README.md` is for contributing information. It needs cleaned up in general. The only relevant content seems to be the Intro, Releases (needs updating), and the first two sentences for "Development: Getting Started". Everything else could go.

The file `README_APP.md` is what is set as the _developing_ information during installation. README.md is deleted and this one is renamed.
- Actually, this is the kind of contributing info that would be helpful; specifically connecting the package `create-redwood-app` to using this repo, and _how_ it uses this repo during install. Also, maybe the contributing info should link to this --> https://community.redwoodjs.com/t/how-to-manually-bootstrap-install-create-redwood-app/115
- The entire "Development" section should likely be deleted or completely overhauled. If we had a "Redwood Quick Start" guide, it would go here. Until then we're better off saying either a) check out the Tutorial or b) read the docs

**2. Language about "Redwood Readiness"**
I'm working in the background on two parts to this: 1) a general disclaimer we'll use in 4 places that links to 2) a separate document explaining, by component, the current state of readiness. In general, Tom wants us to still use the "Redwood is in Alpha" language at this time. But that begs some questions about what's not ready and how we're getting to v1 by the end of the year.

For now, there is immediate confusion to address about Prisma and how it's both "stable" and "experimental". In this PR is makes sense to update the `README_APP.md` file to use this language (which is used in the Framework README):
> WARNING: Redwood is still in the alpha phase of development. We do not recommend that you use Redwood applications in production at this time.

We can avoid the Prisma questions for now. And explain better in a separate document.

**Comment by jtoar** [comment]
@thedavidprice Sorry this one took so long, changes I made/didn't make

- [x] `README.md` General clean up; kept only the relevant content (Intro, Releases, and the first two sentences of "Development: Getting Started")
- [x] `README_APP.md` Don't mention Prisma, instead, update to use the language in the Framework README ("WARNING: Redwood is still in the alpha phase of development...")
- [x] `README_APP.md` Remove "Development" section.
- [ ] `README.md` Update the Release section (wasn't exactly sure what you meant/what this section was for--like releasing a new version of Redwood?)

And I didn't know quite what you meant by this:

> Actually, this is the kind of contributing info that would be helpful; specifically connecting the package create-redwood-app to using this repo, and how it uses this repo during install. Also, maybe the contributing info should link to this --> https://community.redwoodjs.com/t/how-to-manually-bootstrap-install-create-redwood-app/115

If I've got this straight: we need to add content to `README.md` (which is the contributing readme) detailing how to test your contributions as if a user was installing a Redwood app via `yarn create redwood-app ...`?

**Comment by peterp** [comment]
I removed the warning over here: https://github.com/redwoodjs/create-redwood-app/commit/21960fe7cd9c41c1546d239c6950a6db4199b6f5 and added the one in this PR.

**Comment by thedavidprice** [comment]
@jtoar Sorry to sit on this one. Cleaned up and resolved some merge conflicts (I created). 

Merging now.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Redwood
 >👉 **"This is not the Repo you are looking for"** _...most likely_. If your goal is to start building an app using RedwoodJS, you'll want to read through the [Tutorial](https://github.com/redwoodjs/tutorial) and get started from the command line:
 >$ `yarn create redwood-app <directory-name>`
-
+>
 >**This repo is a template used by the RedwoodJS app bootstrap package 'Create Redwood App'**, which is located at [`redwood/packages/create-redwood-app/`](https://github.com/redwoodjs/redwood/tree/main/packages/create-redwood-app). If you're looking to do things like contributing to RedwoodJS development or reference Redwood's full-stack building blocks, then you're in the right place!🌲🎉
```
Comment (thedavidprice) [inline README.md:5]:
```suggestion
**This repo is a template used by the RedwoodJS app bootstrap package 'Create Redwood App'**, which is located at [`redwood/packages/create-redwood-app/`](https://github.com/redwoodjs/redwood/tree/main/packages/create-redwood-app). If you're looking to do things like contributing to RedwoodJS development or reference Redwood's full-stack building blocks, then you're in the right place!🌲🎉
```

File: README_APP.md [hunk 2]
```diff
@@ -23,29 +22,3 @@ yarn redwood dev
 ```
 
 Your browser should open automatically to `http://localhost:8910` to see the web app. Lambda functions run on `http://localhost:8911` and are also proxied to `http://localhost:8910/api/functions/*`.
```
Comment (thedavidprice) [inline README_APP.md:24]:
```suggestion
Your browser should open automatically to `http://localhost:8910` to see the web app. Lambda functions run on `http://localhost:8911` and are also proxied to `http://localhost:8910/.redwood/functions/*`. 
```

**Pull request merged**